bitbots_vision/src/bitbots_vision/vision_modules/yolo_handler.py

```diff
@@ -60,17 +60,17 @@ def __init__(self, config, model_path):
         # Generates a dummy file for the library
         self._generate_dummy_obj_data_file(namepath)
 
-        self.config = config
+        self._config = config
 
         # Setup detector
-        self.net = Detector(bytes(configpath, encoding="utf-8"), bytes(weightpath, encoding="utf-8"), 0.5, bytes(datapath, encoding="utf-8"))
+        self._net = Detector(bytes(configpath, encoding="utf-8"), bytes(weightpath, encoding="utf-8"), 0.5, bytes(datapath, encoding="utf-8"))
         # Set classes
-        self.classes = ["ball", "goalpost"]
+        self._classes = ["ball", "goalpost"]  # TODO Jan: this has no usages?! Why? Remove?
```

Comment:
See here...

Reply:
Indeed this list is a usless development relict. The handler itself uses only the class_id.